utils/build.mjs

```diff
@@ -65,32 +49,49 @@ export const processActionYml = (
     return actionYml;
 };
 
+/**
+ * Copies all files matching the `sourcePath` to the output bundle folder
+ * (dist/). Globs are supported and expanded with fast-glob.
+ */
+const bundleIfExists = (sourcePath) => {
+    for (const fp of fg.globSync(sourcePath)) {
+        const targetPath = path.join(
+            path.dirname(fp),
+            "dist",
+            path.basename(fp),
+        );
+
+        if (fs.existsSync(fp)) {
+            console.log(`  Copying ${fp} to ${targetPath}`);
+            fs.copyFileSync(fp, targetPath);
+        }
+    }
+};
+
 export const buildPackage = (name, packageJsons, monorepoName) => {
-    const dist = `actions/${name}/dist`;
+    const base = `actions/${name}`;
+    const dist = `${base}/dist`;
+
+    // Clean before starting the build
     if (fs.existsSync(dist)) {
         fs.rmSync(dist, {recursive: true});
     }
-    copyDir(`actions/${name}`, dist);
-    const yml = `actions/${name}/dist/action.yml`;
-    const actionYml = fs.readFileSync(yml, "utf8");
+    fs.mkdirSync(dist);
+
+    bundleIfExists(`${base}/package.json`);
+    bundleIfExists(`${base}/*.md`);
```

Comment:
This means that we "opt in" to the files we bundle instead of copying them all. The original code was copying the `node_modules` dirs over which isn't needed because `ncc` creates a single-file bundle that doesn't need any external libs.